src/main/java/fr/nelson/you_are_the_hero/service/RefreshTokenService.java

```diff
@@ -0,0 +1,48 @@
+package fr.nelson.you_are_the_hero.service;
+
+import fr.nelson.you_are_the_hero.model.db.RefreshToken;
+import fr.nelson.you_are_the_hero.repository.RefreshTokenRepository;
+import fr.nelson.you_are_the_hero.utility.JwtUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.time.Instant;
+import java.util.UUID;
+
+@Service
+public class RefreshTokenService {
+
+    @Autowired
+    private RefreshTokenRepository refreshTokenRepository;
+
+    @Autowired
+    private UserService userService;
+
+    @Autowired
+    JwtUtil jwtUtil;
```

Comment:
visibility is missing
and you should use a constructor instead of @autowired (spring preconisation)